tutorials/text/text_intro.py

```diff
@@ -93,3 +96,337 @@
+
+plt.show()
+
+##############################################################################
+# Finally, we can use native TeX rendering in all text objects and have
+# multiple lines:
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(bottom=0.2, left=0.2)
+ax.plot(x1, np.cumsum(y1**2))
+ax.set_xlabel('time [s] \n This was a long experiment')
+ax.set_ylabel('$\int\ Y^2\ dt\ \ [V^2 s]$')
+plt.show()
+
+
+##############################################################################
+# Titles
+# ======
+#
+# Subplot titles are set in much the same way as labels, but there is
+# the *loc* keyword arguments that can change the position and justification
+# from the default value of ``loc=center``.
+
+fig, axs = plt.subplots(3, 1, figsize=(5, 6), tight_layout=True)
+locs = ['center', 'left', 'right']
+for ax, loc in zip(axs, locs):
+    ax.plot(x1, y1)
+    ax.set_title('Title with loc at '+loc, loc=loc)
+plt.show()
+
+##############################################################################
+# Vertical spacing for titles is controlled via ``rcParams[axes.titlepad]``,
+# which defaults to 5 points.  Setting to a different value moves the title.
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(top=0.8)
+ax.plot(x1, y1)
+ax.set_title('Vertically offset title', pad=30)
+plt.show()
+
+
+##############################################################################
+# Ticks and ticklabels
+# ====================
+#
+# Placing ticks and ticklabels is a very tricky aspect of making a figure.
+# Matplotlib does the best it can automatically, but it also offers a very
+# flexible framework for determining the choices for tick locations, and
+# how they are labelled.
+#
+# Terminology
+# ~~~~~~~~~~~
+#
+# *Axes*  (typically) have an `matplotlib.axis` object for the ``ax.xaxis``
+# and ``ax.yaxis`` that
+# contain the information about how the labels in the axis are laid out.
+#
+# The axis API is explained in detail in the documentation to
+# `~matplotlib.axis`.
+#
+# An Axis object has major and minor ticks.  The Axis has a
+# `matplotlib.xaxis.set_major_locator` and
+# `matplotlib.xaxis.set_minor_locator` methods that use the data being plotted
+# to determine
+# the location of major and minor ticks.  There are also
+# `matplotlib.xaxis.set_major_formatter` and
+# `matplotlib.xaxis.set_minor_formatters` methods that format the tick labels.
+#
+# Simple ticks
+# ~~~~~~~~~~~~
+#
+# It often is convenient to simply define the
+# tick values, and sometimes the tick labels, overriding the default
+# locators and formatters.  This is discouraged because it breaks itneractive
+# navigation of the plot.  It also can reset the axis limits: note that
+# the second plot has the ticks we asked for, including ones that are
+# well outside the automatic view limits.
+
+fig, axs = plt.subplots(2, 1, figsize=(5, 3), tight_layout=True)
+axs[0].plot(x1, y1)
+axs[1].plot(x1, y1)
+axs[1].xaxis.set_ticks(np.arange(0., 8.1, 2.))
```

Comment:
The "8.1" is a bit ugh.  In this specific case I'd just write [0, 2, 4, 6, 8] explicitly (or even stop at 6, which is enough for the example).

Reply:
I did 8.1 on purpose so show the bad side effect of using `set_ticks`.  I uses `arange` because thats just something I typically do to get ticks.